Libraries/Components/Touchable/Touchable.js

```diff
@@ -668,9 +668,10 @@ var TouchableMixin = {
       this.touchableHandleActivePressIn && this.touchableHandleActivePressIn(e);
     } else if (!newIsHighlight && curIsHighlight && this.touchableHandleActivePressOut) {
       if (this.touchableGetPressOutDelayMS && this.touchableGetPressOutDelayMS()) {
-        this.pressOutDelayTimeout = this.setTimeout(function() {
```

Comment:
@ide bind is used all over this file, zero arrow functions, surely keep it the same?